app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py

```diff
@@ -403,6 +453,13 @@ def getAvailableBeerAmount(self, player: Player, town: Town) -> int:
                 break
         return amount
 
+    """
+    isIronAvailable
+    
+    """
+    def isIronAvailable(self):
```

Comment:
I'm not sure if we need this? We don't have an equivalent `isCoalAvailable` function. If anything, we'd at least want to pass in a Player's money and check if they can afford the iron